datafusion/core/tests/sqllogictests/test_files/interval.slt

```diff
@@ -432,11 +432,15 @@ select '1 month'::interval + '1980-01-01T12:00:00'::timestamp;
 
 # Exected error: interval (scalar) - date / timestamp (scalar)
 
-query error DataFusion error: type_coercion\ncaused by\nError during planning: Interval\(MonthDayNano\) \- Date32 can't be evaluated because there isn't a common type to coerce the types to
+query D
```

Comment:
Making this work wasn't actually intentional, but I don't see a reason why they shouldn't work